content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md

```diff
@@ -111,6 +111,18 @@ Setting the ingress IP depends on the cluster provider:
     $ export INGRESS_HOST=127.0.0.1
     {{< /text >}}
 
+1.  _minikube:_
```

Comment:
Instead of adding this here, I would suggest adding a tip around line 40, something like this:
```
{{< tip >}}
If you are using minikube, you can easily start an external load balancer (recommended) by running the following command in a different terminal:

{{< text bash >}}
$ minikube tunnel
{{< /text >}}

{{< /tip >}}
```

Reply:
good idea, followed your suggestion with my next commit